src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -43,15 +43,19 @@ export const ChatMessage: React.FC<ChatProps> = ({
 
   const renderedReplies = useMemo(
     () =>
-      replies?.map((reply) => (
-        <div key={reply.id} className="ChatMessage__reply">
-          <RenderMarkdown text={reply.text} allowHeadings={false} />
-          <ChatMessageInfo
-            message={reply}
-            deleteMessage={() => deleteMessage(reply.id)}
-          />
-        </div>
-      )),
+      replies?.map((reply) => {
+        return (
+          <div key={reply.id} className="ChatMessage__reply">
+            <RenderMarkdown text={reply.text} allowHeadings={false} />
+            <ChatMessageInfo
+              message={reply}
+              deleteMessage={
+                deleteMessage ? () => deleteMessage(reply.id) : undefined
```

Comment:
```suggestion
                deleteMessage && () => deleteMessage(reply.id)
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1963/commits/c6b97bed424271dbb74a0353cb2bb89dfb540c05